### PR TITLE
Exception helpers

### PR DESCRIPTION
**Pull request by nmbro**

### CODE REVIEW

File: src/OpenTelemetry.Api/Trace/SpanAttributeConstants.cs [hunk 1]
```diff
@@ -27,5 +27,9 @@ internal static class SpanAttributeConstants
         public static readonly string HttpRequestSizeKey = "http.request.size";
         public static readonly string HttpResponseSizeKey = "http.response.size";
         public static readonly string HttpRouteKey = "http.route";
+
+        public static readonly string ErrorNameKey = "error.name";
```
Comment (lmolkova) [inline src/OpenTelemetry.Api/Trace/SpanAttributeConstants.cs]: are error conventions documented in the spec? I cannot find anything there

Comment (nmbro) [reply src/OpenTelemetry.Api/Trace/SpanAttributeConstants.cs]: I don't think so - but it's a very normal .net convention; and it's supported StackDriver at least

**Comment by MikeGoldsmith** [comment]
There has been recent discussion on how errors are to be handled and likely will form part of the 0.4 OTel spec. I agree it's probably best to hold back on defining these until they're in the spec :+1:

**Comment by MikeGoldsmith** [comment]
@nmbro are you happy for this to be closed for now, and we can re-open once we have spec?

**Comment by nmbro** [comment]
Sure thing

**Pull request closed without merging**